packages/svelte/.prettierrc

```diff
@@ -0,0 +1,16 @@
+{
+  "tabWidth": 2,
+  "useTabs": false,
+  "singleQuote": true,
+  "arrowParens": "avoid",
+  "trailingComma": "all",
+  "plugins": ["prettier-plugin-svelte", "prettier-plugin-tailwindcss"],
+  "overrides": [
+    {
+      "files": "*.svelte",
+      "options": {
+        "parser": "svelte"
+      }
+    }
+  ]
+}
```

Comment:
can we fallback on repo defaults?

Reply:
I was trying for my life to figure that out -- AFAICT Prettier does not support configuration merging, and we can't add this configuration to the repo root, because the prettier plugins only support prettier >= 3.

So basically yes, we can remove this entirely in favor of the repo script -- if we can upgrade to Prettier 3. Let me see if I can go back to `main` and open a PR.

Reply:
Opened PR, closing this